.pre-commit-config.yaml

```diff
@@ -69,6 +69,43 @@ repos:
       - id: ruff
         args: ["--fix"]
 
+  - repo: https://github.com/PyCQA/docformatter
+    rev: v1.5.0
+    hooks:
+      - id: docformatter
+        args: [--in-place]
+        additional_dependencies:
+          - tomli
+        exclude: |
+          (?x)(
+              astropy/__init__\.py |
+              astropy/_dev/ |
+              astropy/config/ |
+              astropy/conftest\.py |
+              astropy/constants/ |
+              astropy/convolution/ |
+              astropy/coordinates/ |
+              astropy/extern/ |
+              astropy/io/ |
+              astropy/logger\.py |
+              astropy/modeling/ |
+              astropy/nddata/ |
+              astropy/samp/ |
+              astropy/stats/ |
+              astropy/table/ |
+              astropy/tests/ |
+              astropy/time/ |
+              astropy/timeseries/ |
+              astropy/uncertainty/ |
+              astropy/units/ |
+              astropy/utils/ |
+              astropy/version\.py |
+              astropy/visualization/ |
+              astropy/wcs/ |
+              docs/ |
+              examples/
```

Comment:
Can this be moved into the pyproject.toml configuration for docformatter?

Reply:
What are your larger thoughts on the tool?

Reply:
We do have to move this configuration to `pyproject.toml`. [`pre-commit`](https://pre-commit.com/) is a framework for automatically running and managing different tools as a part of the committing process, but the tools can also be run independently. If they are fully configured in `.pre-commit-config.yaml` then it might be possible that the tool by itself does not detect any problems but as a part of `pre-commit` it does, or vice versa.

The [`docformatter`](https://pypi.org/project/docformatter/) documentation says that the [command line options can also be stored in a configuration file](https://docformatter.readthedocs.io/en/latest/configuration.html#how-to-configure-docformatter), and the [CLI options are also documented](https://docformatter.readthedocs.io/en/latest/usage.html), so it's not clear to me what would be lacking there.

Reply:
It doesn't appear to work with pre-commit. Maybe to do with how pre-commit feeds files to docformatter.
We ran into something similar with ruff.

Reply:
Maybe just don't run `docformatter` independently of pre-commit.

Reply:
They may need to add something like the force-exclude.

Reply:
IMO we can start with the settings here in pre-commit then try further experimentation to move them to pyproject / upstream this issue.